tests/test_env.py

```diff
@@ -82,28 +84,30 @@ def test_install_fails(tmpdir, caplog):
 def test_install_package_already_installed_in_parent_env(tmpdir):
     """Test using and installing a package that is already present in the parent virtual environment."""
     # get all packages in the parent
-    parent_installed = [
-        p for p in env.VirtualEnv._get_installed_packages(sys.executable).keys() if p not in ["pip", "setuptools"]
-    ]
+    parent_installed = list(env.VirtualEnv._get_installed_packages(sys.executable).keys())
 
     # create a venv and list all packages available in the venv
     venv = env.VirtualEnv(tmpdir)
     venv.use_virtual_env()
 
-    installed_packages = [
-        p for p in env.VirtualEnv._get_installed_packages(venv._parent_python).keys() if p not in ["pip", "setuptools"]
-    ]
+    installed_packages = list(env.VirtualEnv._get_installed_packages(venv._parent_python).keys())
 
     # verify that the venv sees all parent packages
     assert not set(parent_installed) - set(installed_packages)
 
+    # site dir should be empty
+    site_dir = os.path.join(venv.env_path, "lib/python3.6/site-packages")
```

Comment:
Does hardcoding this as python3.6 make sense when we're gradually transitioning to python3.8 support?

Reply:
right, good catch